cleanup_job.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
```

Comment:
Can you add this file somewhere under the `cachito/workers` directory? I imagine we want this bundled as part of the Python package.

Also, could you add an entry in `console_scripts` in `setup.py`:
https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html#the-console-scripts-entry-point

That way you can just call it with `cachito-cleanup` or something like that.